### DISCUSSION TITLE
4.6.6 beta 3 support [rtx 40xx super support]

### DISCUSSION THREAD

**Discussion by Maxim766** [ts-korhonen/LibreHardwareMonitorAfterburnerPlugin#14] [Ideas]
Hello, @ts-korhonen 

Are you planning to update plugin for new version of MSI Afterburner?

**Comment by ts-korhonen** [comment]
Hi!

A new release is usually verified against the latest release of Afterburner at the time, which is mentioned in readme and release notes, but there is no strict dependency between the versions. There haven't been breaking changes on the plugin interface so in reality there is compatibility for a wide range of versions. I don't see such changes in the 4.6.6 beta 3 either - can you explain the problem you are having with the version?

**Comment by Maxim766** [comment]
Hi!

When I choose any options for GPU in setup menu in your plugin - nothing appears in monitoring graphs menu. I guess because I have 4070 ti super - support for 40 super cards has been added into beta versions only.

**Reply by ts-korhonen** [reply]
My GPU is also a RTX 4070 Ti Super. I just installed the Afterburner 4.6.6 beta 3 and it works just like it does with 4.6.5.

Is there a log file (LibreHardwareMonitor.dll.log) in the folder you placed the plugin in?

**Reply by Maxim766** [reply]
I forgot to mention which version I have used. 0.5 is my case. Installed 0.6 - it's working now. 

But which temperature sensors for your GPU you have? I have only GPU core and GPU hotspot. In GPU-Z and HWiNFO also presented memory temperature. Is this possible to implement this sensor into the plugin?

Thx in advance

**Reply by ts-korhonen** [reply]
> But which temperature sensors for your GPU you have? I have only GPU core and GPU hotspot. 

Those are the only ones I have as well.

> In GPU-Z and HWiNFO also presented memory temperature. Is this possible to implement this sensor into the plugin?

This plugin is only a wrapper for the LibreHardwareMonitorLib which is providing the sensor data. Support for the device needs to be implemented in [LibreHardwareMonitor](https://github.com/LibreHardwareMonitor/LibreHardwareMonitor) to appear in the plugin.

**Comment by ts-korhonen** [comment]
Release v0.7 has support for RTX 40 Super series.
